Makefile

```diff
@@ -10,6 +10,7 @@ template:
 	$(container_cmd) run -ti --rm -v $(mkfile_path):/apps docker://alpine/helm:3.11.1 template spire-flex 
 
 test:
+	mkdir -p spire-flex/tests/__snapshot__
```

Comment:
```suggestion
```
suggesting removal of the makefile directory creation

Reply:
I can't figure out how to suggest the adding for a spire-flex/tests/__snapshot__/.empty_dir file.  Please add it if and when you remove the Makefile mkdir command.